### ISSUE TITLE
No proper instruction!

### ISSUE THREAD

**Issue by SylarRuby** [Shopify/shopify_theme#210]
Where is the instruction?

> The Shopify theme gem is a command line tool that allows you to manage and edit theme files directly on your computer. This allows you to use your preferred code editing software opposed to the built in code editor available in the “Edit HTML/CSS” page of your store. Files can be manually or automatically uploaded, downloaded, and synced. Depending on how you develop themes and your personal preference, this can allow for a more enjoyable and streamlined workflow. If the command line is scary for you, check out the Desktop Theme Editor app.

Where is step one! Where do you get the command `theme` from? BS!

**Comment by veganista** [comment]
You should be able to install the gem uisng `gem install install_theme`

**Comment by jamesMcMeex** [comment]
Yeah, it's odd that the actual install command is not in the readme... However it's a Ruby gem so you can find it easily enough by searching https://rubygems.org/ where you can find the command.

It's `gem install shopify_theme` btw :)

**Comment by tbeseda** [comment]
The link at the top of this repo has a section **"Installation"**...
http://shopify.github.io/shopify_theme/
